lib/eapeak/parse.py

```diff
@@ -983,23 +904,145 @@ def cursesScreenDrawHandler(self, save_to_xml):
+						the_cheese_stands_alone = False
+					messages.append((TAB_DEPTH_2, piece + ': ' + network.wpsData[piece]))
+			if not the_cheese_stands_alone:
+				messages.append(CURSES_LINE_BREAK)
+			del the_cheese_stands_alone, piece # pylint: disable=undefined-loop-variable
+		if network.clients:
+			messages.append((TAB_LENGTH, 'Clients:         '))
+			clients = network.clients.values()
+			for i in range(0, len(clients)):
+				client = clients[i]
+				messages.append((TAB_DEPTH_2, 'Client ' + str(i + 1) + ') MAC: ' + client.mac))
+				if client.eapTypes:
+					self.get_client_eap(client, messages)
+				else:
+					messages.append((TAB_DEPTH_2, 'EAP Types: [ UNKNOWN ]'))
+				if client.identities:
+					messages.append((TAB_DEPTH_2, 'Identities:'))
+				for ident, eap in client.identities.items():
+					messages.append((TAB_DEPTH_3, '(' + EAP_TYPES[eap] + ') ' + ident))
+				if client.mschap:
+					first = True
+					for value in client.mschap:
+						if 'r' not in value:
+							continue
+						if first:
+							messages.append((TAB_DEPTH_2, 'MSChap:'))
+							first = False
+						messages.append((TAB_DEPTH_3, 'EAP Type: ' + EAP_TYPES[value['t']] + ', Identity: ' + value['i']))
+						messages.append((TAB_DEPTH_3, 'C: ' + value['c']))
+						messages.append((TAB_DEPTH_3, 'R: ' + value['r']))
+					del first
+				if client.wpsData:
+					the_cheese_stands_alone = True
+					for piece in ['Manufacturer', 'Model Name', 'Model Number', 'Device Name']:
+						if client.wpsData.has_key(piece):
+							if the_cheese_stands_alone:
+								messages.append((TAB_DEPTH_2, 'WPS Information:'))
+								the_cheese_stands_alone = False
+							messages.append((TAB_DEPTH_3, piece + ': ' + client.wpsData[piece]))
+					del the_cheese_stands_alone, piece # pylint: disable=undefined-loop-variable
+				messages.append(CURSES_LINE_BREAK)
+			messages.pop()	# trash the trailing line break
+			del clients # pylint: disable=undefined-loop-variable
+		else:
+			messages.append((TAB_LENGTH, 'Clients: [ NONE ]'))
+
+	def get_network_info(self, messages, network):
+		tmpEapTypes = []
+		if network.eapTypes:
+			for eType in network.eapTypes:
+				if eType in EAP_TYPES:
+					tmpEapTypes.append(EAP_TYPES[eType])
+				else:
+					tmpEapTypes.append(str(eType))
+		if tmpEapTypes:
+			messages.append((TAB_LENGTH, 'EAP Types: ' + ", ".join(tmpEapTypes)))
+		else:
+			messages.append((TAB_LENGTH, 'EAP Types: [ NONE ]'))
+		tmpVendorIDs = []
+		if network.expandedVendorIDs:
+			for vType in network.expandedVendorIDs:
+				if vType in EXPANDED_EAP_VENDOR_IDS:
+					tmpVendorIDs.append(EXPANDED_EAP_VENDOR_IDS[vType])
+				else:
+					tmpVendorIDs.append(str(vType))
+		if tmpVendorIDs:
+			messages.append((TAB_LENGTH, 'Expanded Vendor IDs: ' + ", ".join(tmpVendorIDs)))
+		del tmpEapTypes, tmpVendorIDs
+
+	def set_max_offset(self, max_offset):
+		if max_offset < 0:
+			max_offset = 0
+		if self.curses_row_offset > max_offset:
+			self.curses_row_offset = max_offset
+		del max_offset
```

Comment:
Single blank line after this and it doesn't look necessary to delete `max_offset` since the function just returns.